lib/scout_apm/auto_instrument/instruction_sequence.rb

```diff
@@ -0,0 +1,22 @@
+
+require 'scout_apm/auto_instrument/rails'
+
+module ScoutApm
+  module AutoInstrument
+    module InstructionSequence
+      CONTROLLER_PATH_PATTERN = /\/apm\/app\/controllers\/.*_controller.rb$/
+
+      def load_iseq(path)
+        if path =~ CONTROLLER_PATH_PATTERN
+          new_code = Rails.rewrite(path)
+
+          return compile(new_code, path, filepath)
```

Comment:
`filepath` is undefined. Also, I think the arguments to `compile` are wrong - [https://ruby-doc.org/core-2.5.0/RubyVM/InstructionSequence.html#method-c-compile](https://ruby-doc.org/core-2.5.0/RubyVM/InstructionSequence.html#method-c-compile)